packages/cli/vite.config.ts

```diff
@@ -0,0 +1,26 @@
+/// <reference types='vitest' />
+import { defineConfig } from 'vite';
+
+import { nxViteTsPaths } from '@nx/vite/plugins/nx-tsconfig-paths.plugin';
+
+export default defineConfig({
+  root: __dirname,
+  cacheDir: '../../node_modules/.vite/packages/cli',
+
+  plugins: [nxViteTsPaths()],
+
+  test: {
+    globals: true,
+    cache: {
+      dir: '../../node_modules/.vitest',
+    },
+    environment: 'node',
+    include: ['src/**/*.test.ts'],
+
+    reporters: ['default'],
```

Comment:
Note: For me personally (working on Windows) after updating Nx to 17 the default reporter created a lot of mess and we had to switch to `basic`. But hopefully that's just me and this works just fine for you.